cluster/addons/rbac/kubelet-certificate-management.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: beta:gce:kubelet-certificate-bootstrap
```

Comment:
I don't feel strongly, though I would maybe flip the order (`gce:` prefix first)